README.md

```diff
@@ -297,6 +297,7 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
 We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
 
 + Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the IP of the edge node
++ Make sure role is set to edge for the node. For this a key of the form `"node-role.kubernetes.io/edge"` must be present in `labels` tag of `metadata`. If role is not set for the node pods, configmap and secrets cannot sync with the node.
```

Comment:
Make sure role is set to edge for the node. For this a key of the form `"node-role.kubernetes.io/edge"` must be present in `labels` tag of `metadata`. ->

Please ensure to add the role `node-role.kubernetes.io/edge` to the `build/node.json` file.
< give the node.json example here >

If the role is not set for the node , the pods, configmaps and secrets created/updated in the cloud cannot be synced with the node they are targeted for.